project/justfix_environment.py

```diff
@@ -350,6 +350,15 @@ class JustfixEnvironment(typed_environ.BaseEnvironment):
     # works-in-progress (WIPs), i.e. only partially localized.
     ENABLE_WIP_LOCALES: bool = False
 
+    # A comma-separated list of hostname redirects. Each redirect
+    # is of the form "<src> to <dest>", where src is the hostname to redirect
+    # from, and where dest is the hostname to redirect to.  If any requests
```

Comment:
Ooooh, I like how conversational the syntax is here. It wouldn't be my instinct to configure something as a string "foo.com to bar.com" but you have great documentation of it everywhere so I like it.